lib/exchange/adapters/in_memory_time_series.ex

```diff
@@ -199,7 +187,8 @@ defmodule Exchange.Adapters.InMemoryTimeSeries do
   end
 
   @spec get_live_orders(ticker :: atom) :: [Exchange.Order]
-  def get_live_orders(_ticker) do
-    []
+  def get_live_orders(ticker) do
+    GenServer.call(:in_memory_time_series, {:live_orders, ticker})
+    |> elem(1)
```

Comment:
the `elem` notation is a bit obscure (if someone else reads the code they won't know what you're returning).
would be better to pattern match on the GenServer response and when `{:ok, orders}` is received, return `orders`